lib/getServerSideProps.ts

```diff
@@ -18,7 +20,7 @@ export async function getServerSideProps(context: GetSessionParams) {
   // If the user is logged in, simply return the session.
   return {
     props: {
-      session,
+      session: JSON.parse(JSON.stringify(session)),
```

Comment:
does passing `session` directly not work?

Reply:
There will be a serialization error if passed directly.